embedded-workshop-book/src/parts-embedded-program.md

```diff
@@ -1,26 +1,35 @@
 # Parts of an Embedded Program
 
-[❗️Intro ]
-Open the `beginner/apps` folder in VS Code.
+We will look at the elements that distinguish an embedded Rust program from a desktop program.
+
+
+✅ Open the `beginner/apps` folder in VS Code.
 
 ``` console
 $ # or use "File > Open Folder" in VS Code
 $ code beginner/apps
 ```
 
-Then open the `src/bin/hello.rs` file.
+✅ Then open the `src/bin/hello.rs` file.
+
+
 
-If you find it more convenient to open the repository at the root then please also add the `beginner/apps` folder to the VS Code workspace: right click the left side panel, select "Add folder to workspace" and add the `beginner/apps` folder.
-[❗️No optional way of doing things here]
+> NOTE: If you find it more convenient to open the repository at the root then please also add the `beginner/apps` folder to the VS Code workspace: right click the left side panel, select "Add folder to workspace" and add the `beginner/apps` folder.
```

Comment:
could you please remove this NOTE? Unfortunately, doing this breaks Rust Analyzer. We meant to remove all references to creating multi-folder workspaces but missed this one.

Reply:
sure!